tests/Handler/ActionHandlerTest.php

```diff
@@ -1,11 +1,11 @@
 <?php
 namespace EquipTests\Handler;
 
+use EquipTests\Configuration\ConfigurationTestCase;
+use EquipTests\Fake\FakeDomain;
```

Comment:
Actually these were in the right place. `Equip\` sorts lower than `EquipTests\`.